test/unit/postprocessing/test_plots.py

```diff
@@ -31,6 +25,8 @@
 
 
 def _fit_and_plot(constraints, plotting_data):
+    import matplotlib.pyplot as plt
```

Comment:
Doesn't flake8 scream about an import which isn't at the top of the file?

Reply:
Interestingly, it doesn't!